src/js/load.js

```diff
@@ -40,13 +40,13 @@ export default function configureApplication(profileId, config) {
         config.analytics.registerEvents(controller);
 
     const mapcontrol = new MapControl(config);
-    mapcontrol.popup = new Popup(mapcontrol.map);
+    mapcontrol.popup = new Popup(config, mapcontrol.map);
     const pointData = new PointData(api, mapcontrol.map, profileId, config);
     const pointDataTray = new PointDataTray(api, profileId);
     const mapchip = new MapChip(config.choropleth);
     const search = new Search(api, profileId, 2);
     const profileLoader = new ProfileLoader(config, api, profileId);
```

Comment:
this should be changed to the `formattingConfig` the profileloader does not use config in a any other way.

Reply:
@emre2038 can you fix this one as well? thanks.